cli/commands/benchmark-web-vitals.mjs

```diff
@@ -67,16 +67,22 @@ export const options = [
 		argname: '-t, --throttle-cpu <factor>',
 		description: 'Enable CPU throttling to emulate slow CPUs',
 	},
+	{
+		argname: '-c, --network-condition <predefined>',
+		description:
+			'Enable emulation of network conditions (may be either "Slow 3G" or "Fast 3G")',
+	},
 ];
 
 /**
  * @typedef Params
- * @property {string}  output            - See above.
- * @property {number}  amount            - See above.
- * @property {?string} file              - See above.
- * @property {boolean} showPercentiles   - See above.
- * @property {?number} cpuThrottleFactor - See above.
- * @property {?string} url               - See above.
+ * @property {?string}                url               - See above.
+ * @property {number}                 amount            - See above.
+ * @property {?string}                file              - See above.
+ * @property {string}                 output            - See above.
+ * @property {boolean}                showPercentiles   - See above.
+ * @property {?number}                cpuThrottleFactor - See above.
+ * @property {?("Slow 3G"|"Fast 3G")} networkCondition  - See above.
```

Comment:
Is this "enum" like annotation valid? Just checking since I've never seen it before :)

Reply:
Humm. Well, [it is in TypeScript](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types), but it's apparently [not recognized at least by the TypeScript compiler](https://www.typescriptlang.org/play?#code/DYUwLgBAxg9gdgEwIwC4IAoBEAxAhgZ0gGYBxTAH0wGVgYB3CUiuAV2GAEoBuAKFkSQQAvBBwFiZXv2TDRNeo0l94Mka3ZSVgkZgDCuAEahMXCAHozEQKDkEAKIAnezHsQCEEAA8ADiChgQCAB0PDxmAFRhEAACYACePhAA3lh4hIoU1LQMTOTqnAC+EGFmPKCQ0gBMmogVsmJpTNUItTry2UqVsnlNLaL6RiAm5paAMuQQAHIw7o7OADQQcDCQnj5+AcFAA) even though [it's supposed to be](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html).

Reply:
Removed in 2a1d735.

Reply:
> Humm. Well, [it is in TypeScript](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types), but it's apparently [not recognized at least by the TypeScript compiler](https://www.typescriptlang.org/play?#code/DYUwLgBAxg9gdgEwIwC4IAoBEAxAhgZ0gGYBxTAH0wGVgYB3CUiuAV2GAEoBuAKFkSQQAvBBwFiZXv2TDRNeo0l94Mka3ZSVgkZgDCuAEahMXCAHozEQKDkEAKIAnezHsQCEEAA8ADiChgQCAB0PDxmAFRhEAACYACePhAA3lh4hIoU1LQMTOTqnAC+EGFmPKCQ0gBMmogVsmJpTNUItTry2UqVsnlNLaL6RiAm5paAMuQQAHIw7o7OADQQcDCQnj5+AcFAA) even though [it's supposed to be](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html).

It does work :-) You just need to switch the language from TS to JS in the playground config.

`{?("Slow 3G"|"Fast 3G")}` is definitely valid, so I'd suggest reverting 2a1d7357ecad69fe6d77db28cfda91fa1b2ca88f

Reply:
@swissspidy What are you referring to with "playground config"?

Reply:
It [works](https://www.typescriptlang.org/play?filetype=js#code/PQKhAIAEBcE8AcCm4DeAKARAMQIYGdpwBmAcQwB8MBlAGwHsB3Ys8gOwFcaaBKAX3BDAAUDUSEAxnVYATAEwBuIZJmzwAXnDZ8hUhkXK56zbUbM9SqYY0cu+y6o0YAwjgBGoveGDBwgUHJwAKIATkF0QQA04PjgiAAeSOLQiNIAdEJAA)!

> ![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/ac604889-771e-473d-b7a0-fd11806a8515)

> @swissspidy What are you referring to with "playground config"?

@felixarntz right here:

> ![Screenshot 2023-08-11 13 09 59](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/b7eafc72-cc93-4600-a91d-eaf78294e6dd)

But is this specific to the TypeScript compiler's parsing of JSDoc? Is it valid of JSDoc generally?

Reply:
Regardless, VS Code seems to know what's going on:

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/31c05845-7d28-47db-a12d-f598b4dfdb41)

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/1d8c234a-e625-4d43-88b5-99e300997d19)

Reply:
And PhpStorm does too:

![Screenshot 2023-08-11 13 29 59](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/9fb68a4c-df3e-4a8c-a083-c92d66b208cf)

![Screenshot 2023-08-11 13 30 30](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/d5364f8d-df79-491b-bfb6-396a3d80b561)

![Screenshot 2023-08-11 13 34 44](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/68f24163-e74d-494d-b984-f16a7b978a2f)

Reply:
> But is this specific to the TypeScript compiler's parsing of JSDoc?

Yes. It's quite common to use TypeScript for JSDoc as it's more powerful.

> Is it valid of JSDoc generally?

No. See https://github.com/jsdoc/jsdoc/issues/629